docs/cugraph/source/nx_cugraph/nx_cugraph.md

```diff
@@ -44,25 +43,25 @@ user@machine:/# ipython bc_demo.ipy
 
 You will observe a run time of approximately 7 minutes...more or less depending on your cpu.
 
-Run the command again, this time specifiying cugraph as the NetworkX backend of choice.
+Run the command again, this time specifiying cugraph as the NetworkX backend.
 ```
 user@machine:/# NETWORKX_BACKEND_PRIORITY=cugraph ipython bc_demo.ipy
 ```
 This run will be much faster, typically around 20 seconds depending on your GPU.
 ```
 user@machine:/# NETWORKX_BACKEND_PRIORITY=cugraph ipython bc_demo.ipy
 ```
-There is also an option to add caching. This will dramatically help performance when running multiple algorithms on the same graph.
+There is also an option to cache the graph conversion to GPU. This can dramatically improve performance when running multiple algorithms on the same graph.
 ```
-NETWORKX_BACKEND_PRIORITY=cugraph CACHE_CONVERTED_GRAPH=True ipython bc_demo.ipy
+NETWORKX_BACKEND_PRIORITY=cugraph CACHE_CONVERTED_GRAPHS=True ipython bc_demo.ipy
 ```
 
-When running Python interactively, cugraph backend can be specified as an argument in the algorithm call.
+When running Python interactively, the cugraph backend can be specified as an argument in the algorithm call.
 
 For example:
 ```
 nx.betweenness_centrality(cit_patents_graph, k=k, backend="cugraph")
 ```
 
 
-The latest list of algorithms that can be dispatched to nx-cuGraph for acceleration is found [here](https://github.com/rapidsai/cugraph/blob/main/python/nx-cugraph/README.md#algorithms).
+The latest list of algorithms supported by nx-cuGraph can be found [here](https://github.com/rapidsai/cugraph/blob/main/python/nx-cugraph/README.md#algorithms).
```

Comment:
Should we avoid nx-cuGraph? I think this capitalization is kinda weird.
```suggestion
The latest list of algorithms supported by nx-cugraph can be found [here](https://github.com/rapidsai/cugraph/blob/main/python/nx-cugraph/README.md#algorithms).
```

Reply:
Agreed.  I don't think we should ever use nx-cuGraph IMO